src/main/java/net/atos/zac/zoeken/ZoekenService.java

```diff
@@ -159,12 +179,19 @@ private void applyAllowedZaaktypenPolicy(final SolrQuery query) {
         final Set<String> allowedZaaktypen = policyService.getAllowedZaaktypen();
         if (allowedZaaktypen != null) {
             if (allowedZaaktypen.isEmpty()) {
-                query.addFilterQuery(format("%s:\"%s\"", ZAAKTYPE_OMSCHRIJVING_VELD, NON_EXISTING_ZAAKTYPE));
+                query.addFilterQuery(format("%s:%s", ZAAKTYPE_OMSCHRIJVING_VELD, NON_EXISTING_ZAAKTYPE));
             } else {
                 query.addFilterQuery(allowedZaaktypen.stream()
-                                             .map(zaaktype -> format("%s:\"%s\"", ZAAKTYPE_OMSCHRIJVING_VELD, zaaktype))
+                                             .map(ZoekenService::quoted)
+                                             .map(zaaktype -> format("%s:%s", ZAAKTYPE_OMSCHRIJVING_VELD, zaaktype))
                                              .collect(joining(" OR ")));
             }
         }
     }
+
+    private static String quoted(final String waarde) {return "\"" + encoded(waarde) + "\"";}
```

Comment:
Zou je kort kunnen omschrijven (javadoc) wat deze drie methodes doen, in: dit is een "string"  uit: "dit is een \"string"\" zoiets

Reply:
OK